sktime/forecasting/structural.py

```diff
@@ -267,6 +271,9 @@ def __init__(
         self.flags = flags
         self.low_memory = low_memory
 
+        # initialize_default params
+        self.approximate_diffuse_variance = approximate_diffuse_variance
```

Comment:
`approximate_diffuse_variance ` is not used it seems?

Reply:
Sorry. I thought I reverted this file but somehow those lines were still here. I have reverted them and double chcked. They should be gone now.